src/components/BorrowerProfile/BorrowerSideSheetContent.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="tw-bg-eco-green-1 tw--mx-2">
+	<div class="tw--mx-2" style="background-color: rgb(245 245 245)">
```

Comment:
Do we not have a `tw-` class for [this under the "static" tab](https://main--608b4cf87f686c00213841b1.chromatic.com/?path=/story/base-styling-primitives--primitives)?

If not, we should push back and get this assigned a color class.

Reply:
That's a good call out, I forgot this section had a custom bg color.

@The-Best-Joke let's use `tw-bg-secondary`